src/stylesheets/components/_accordions.scss

```diff
@@ -1,16 +1,13 @@
 // Variables
 
-$accordion-border: 3px solid $color-gray-lightest;
+$accordion-border: 3px solid color('base-lighter');
+$text-color: color('base-darkest');
```

Comment:
See this https://github.com/uswds/uswds/pull/2722#discussion_r215398212, we already have a variable for text color, so it's not needed.

Reply:
oh ok, but we do still need a call to the `color` fn for `base-lighter`? Or is there a variable for that too?

Reply:
Good question, I think `$color-theme-base-lighter` may work as well. I'm just looking through the variables file. I don't know if there's a preference for fn vs the variable.

Reply:
Yes, `$color-theme-base-lighter` will work

Reply:
@dano Essentially then, the pattern is:

1. Always default to a theme variable if it exists
2. If not, use the string key of the token (not sure if that is the right word here)

Reply:
Because, I see, for example, in @maya's PR [here](https://github.com/uswds/uswds/pull/2720/files) that she is using 'base-lighter' (the string) vs the variable/

Reply:
I think this is not yet settled law, but it could be a good start — 
1. if the variable is sufficient, use the variable
2. else if the function is sufficient, use the function
3. else use the mixin

I'm hesitant to carve this in stone right now, but it's a useful decision tree

Reply:
0. If a standard rule is sufficient, use a standard rule

Reply:
Super helpful, thanks! I feel like arriving at a decision soon is fine, I'd rather know exactly what the accepted style is, so we can maintain consistency among contributors